### PR TITLE
Firesale fixes for production

### LINKED ISSUE #1777: Task assigning an address without an account, breaks loading the task route
## Steps to reproduce

1. Create a Colony
2. Create a Task
3. Assign as worker to that task, a wallet address that hasn't claimed a profile yet

## Expected behaviour

* It should allow you to add that non-claimed address as a worker, it should work fine to make the payout

## Actual behaviour

* A bunch of components break:

Screenshot from 2019-08-29 14-47-06

### PR DESCRIPTION
**Pull request by rdig**
## Description

This PR was used to fix, then deploy a bunch of fixes to issues that were causing production to break.

**Changes**

- [x] Fix `TaskEventFeed` proper defaults for the user selector _(this was the culprit that was breaking the tasks)_
- [x] `InboxContainer` import proper inbox items record
- [x] `InboxItem` fix relative links _(now that we are using a Popover, the base path is not static / known anymore)_
- [x] `InboxItem` remove hidden overflow to be able to display the `InfoPopover` compoent

Resolves #1777

### CODE REVIEW

File: src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const allReadActions = {
 };
 
 const InboxContainer = ({ full, close }: Props) => {
-  const inboxItems = useSelector(inboxItemsSelector);
+  const { record: inboxItems } = useSelector(inboxItemsSelector);
```
Comment (sprusr) [inline src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx]: Yeah actually here's a case where we don't convert to a JS object I think? Appears to work fine in this component though.

Comment (chmanie) [reply src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx]: I always thought `useSelector` would do the conversion and pick the `record`

Comment (sprusr) [reply src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx]: Oh wow it does! Okay that's really confused me 😵 the react-redux hook is also called `useSelector` (and that doesn't do any extra processing).

Comment (chmanie) [reply src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx]: So why doesn't it do it in this case?

Comment (sprusr) [reply src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx]: No it does, I was thinking `useSelector` was the `react-redux` one.

Comment (chmanie) [reply src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx]: But then why do we need to do it here? Is it a nested `record` for some reason?

Comment (sprusr) [reply src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx]: Oh right, no it doesn't pick `record` just does the `toJS` conversation. Sorry, I was reading/replying too fast!

**Pull request merged**